complete/lib.h

```diff
@@ -35,18 +35,25 @@
 #ifndef MUTT_COMPLETE_LIB_H
 #define MUTT_COMPLETE_LIB_H
 
+#include <stddef.h>
 #include <stdbool.h>
 // IWYU pragma: begin_keep
+#include "compapi.h"
 #include "data.h"
 // IWYU pragma: end_keep
 
 struct Buffer;
 
+extern const struct CompleteOps CompleteCommandOps;
```

Comment:
why are these needed here?
if we use the `#include "compapi.h", are they not defined already (or is this an alias)?

Reply:
`compapi.h` defines the _interface_,
`CompleteCommandOps`, `CompleteLabelOps`, define _implementations_ which are used outside of the library.

We could do with better homes for these.

Reply:
I see. I wasn't aware that C implicitly declares functions as `extern`, and thus got confused by the mix in interface & implementation.